packages/storybook/package.json

```diff
@@ -14,6 +14,8 @@
     "@babel/preset-flow": "^7.22.5",
     "@babel/preset-react": "^7.22.5",
     "@faker-js/faker": "^8.0.2",
+    "@peripleo/maplibre": "^0.3.1",
```

Comment:
Should we also add these dependencies to the `geospatial` package?

Reply:
Of course - done!